src/ServiceControl.Monitoring.SmokeTests.SQS/EndpointTemplates/TestSuiteConstraints.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace ServiceControl.Monitoring.SmokeTests.SQS.EndpointTemplates
+{
+    using NServiceBus.AcceptanceTesting.Support;
+
+    public interface ITestSuiteConstraints
+    {
+        bool SupportsDtc { get; }
+
+        bool SupportsCrossQueueTransactions { get; }
+
+        bool SupportsNativePubSub { get; }
+
+        bool SupportsNativeDeferral { get; }
+
+        bool SupportsOutbox { get; }
+
+        IConfigureEndpointTestExecution TransportConfiguration { get; }
+
+        IConfigureEndpointTestExecution PersistenceConfiguration { get; }
+    }
+
+    // ReSharper disable once PartialTypeWithSinglePart
+    public class TestSuiteConstraints : ITestSuiteConstraints
```

Comment:
Should we align this with https://github.com/Particular/NServiceBus.AmazonSQS/blob/develop/src/AcceptanceTests/TestSuiteConstraints.cs ? I think we should